app/views/shared/head/_lite_mode_css.html.erb

```diff
@@ -14,6 +14,14 @@
   body img {
     display: none;
   }
+	
+	body#podcast .subscribe a.u-url {
+		display: block;
+	}
+	
+	body#podcast .subscribe a.u-url + br {
+		display: none;
```

Comment:
I'm curious why this is done in CSS and not as another `if lite_mode?` in the view. What was your thinking on that?

Reply:
I thought that through css would be a cleaner way to hide the br tag as opposed to logic in the view. I can do it the other way, if that's what you'd prefer.

Reply:
Either way. I was just curious. Let's go with the one that you think is best!